### ISSUE TITLE
Feature - Export lineage in a format that is SQL friendly

### ISSUE THREAD

**Issue by dvasdekis** [reata/sqllineage#298]
Hi again reata, and thanks again for all the help!
Given that we use our database for everything, including filling out our documentation, it makes sense to store the exported lineage inside it.
I would like to have an output option in the CLI that creates a `.sql` file for ingest by our database, instead of the current text-based output format. 
With your permission I'd like to start work on this feature.
Thank you!

**Comment by dvasdekis** [comment]
Just realised that the output is already converted to JSON as part of `drawing.py`, and we can easily work with that.
Much easier instead then to request that a `--json` flag is added to output. The DB can then ingest this.

**Comment by reata** [comment]
Can you elaborate on "creates a .sql file for ingest by our database"? I'm confused with sql. Are you going to create a table with one json column to store all the lineage?

Overall I'm not against export the lineage to json, or graphml. That's what the [io](https://github.com/reata/sqllineage/blob/master/sqllineage/io.py) was designed for. But to_cytoscape is not exporting the lineage to a meaningful json format as it's specifically used by frontend library cytoscape. It's better we redesign the JSON structure.

**Comment by dvasdekis** [comment]
For my use case, it's easy to ingest the JSON generated by `to_cytoscape()` into the database, so it satisfies my need with a minimum of code.  We can query this exported JSON in the cytoscape format via SQL to display the lineage of a datasource in our CKAN data catalog, which we write out as a build-time job (instead of having an ongoing flask server running to display the lineage).

Eventually I also hope to contribute a way to write a static site (hostable on S3/blob storage) that can display the lineage instead of the Flask-based approach.
Thanks!

**Comment by reata** [comment]
OK, I think I got your point. Is it safe for me to assume that you just need one way to serialize the lineage result and then store it somewhere, with the capability to deserialize it later? I believe format is not your concern?

**Comment by dvasdekis** [comment]
> OK, I think I got your point. Is it safe for me to assume that you just need one way to serialize the lineage result and then store it somewhere, with the capability to deserialize it later?

Yes exactly :) JSON libraries are pretty much everywhere. What would be the next step from your perspective?

**Comment by reata** [comment]
Let's go with JSON, but do not use to_cytoscape function. We should design a JSON structure to describe the lineage in a meaningful way. In the long run, that json can also be passed to frontend and then we transform it to cytoscape format there.
